pyproject.toml

```diff
@@ -38,7 +38,6 @@ lint.select = [
 lint.ignore = [
   'A001',
   'A003',
-  'ANN101',
```

Comment:
warning: The following rules have been removed and ignoring them has no effect:
    - ANN101